_data/chains/eip155-1089.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "Humans.ai Mainnet",
+  "chain": "Humans",
+  "rpc": [
+    "https://jsonrpc.humans.nodestake.top/",
+    "https://humans-mainnet-evm.itrocket.net:443",
+    "https://humans-evm-rpc.staketab.org:443",
+    "https://evm.humans.stakepool.dev.br",
+    "https://mainnet-humans-evm.konsortech.xyz",
+    "https://evm-rpc.mainnet.humans.zone/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "HEART",
+    "symbol": "HEART",
+    "decimals": 18
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://humans.ai",
+  "shortName": "humans",
+  "chainId": 1089,
+  "networkId": 1089,
+  "icon": "humans",
+  "explorers": [
+    {
+      "name": "explorer.guru",
+      "url": "https://humans.explorers.guru",
+      "icon": "humans",
+      "standard": "EIP3091"
```

```suggestion
      "standard": "none"
```

explorer does not seem EIP3091 compatible
```suggestion
      "standard": "none"
```

---

thank you @ligi, updated accordingly